File: bots/csharpClient/Bots/Player.cs
using System.Text.Json.Serialization;

namespace CsClient.Bots
{
    public class Player
    {
        [JsonPropertyName("height")]
        public float Height { get; set; }

        [JsonPropertyName("width")]
        public float Width { get; set; }

        [JsonPropertyName("pos_x")]
        public float PosX { get; set; }

        [JsonPropertyName("pos_y")]
        public float PosY { get; set; }

        [JsonPropertyName("rotation")]
        public float Rotation { get; set; }

        [JsonPropertyName("state")]
        public string State { get; set; }

        [JsonConstructor]
        public Player(float height, float width, float posX, float posY, float rotation, string state)
        {
            Height = height;
            Width = width;
            PosX = posX;
            PosY = posY;
            Rotation = rotation;
            State = state;
        }
    }
}

File: bots/kotlinClient/src/main/kotlin/cb/flappybuddy/game/Player.kt
package cb.flappybuddy.game

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class Player(
    @SerialName("height") val height: Double,
    @SerialName("width") val width: Double,
    @SerialName("pos_x") val posX: Double,
    @SerialName("pos_y") val posY: Double,
    @SerialName("rotation") val rotation: Double,
    @SerialName("state") val state: String,
)
